content/articles/using-ecs-to-deploy-docker-app-to-awws/index.md

```diff
@@ -0,0 +1,232 @@
+
+### Creating the flask app
+To create the flask app, we need to install `flask`. In the directory of the project, run the command below:
+```bash
+pip install flask
+```
+Next, create two files `requirements.txt` to hold the project libraries and dependencies and `app.py`. 
+Add the scripts below to the `app.py` file:
+```py
+from flask import Flask, render_template
+import flask
+app = Flask(__name__)
+
+# Index route
+@app.route('/')
+def index():
+    return render_template('index.html')
+    # We are telling flask to look for a file named index in the templates folder
+    #then render it to the user interface. Note that by default flask looks for templates in a folder named templates.
+
+if __name__ == '__main__':
+    app.run(debug=True)
+```
+Create a folder named  `templates` which contains the HTML files rendered on the webpage. In the folder, create a file named `index.html` the add the snippets below.
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Docker | AWS Index</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+</head>
+<body>
+    <div class="container">
+    <div class="jumbotron text-center">
+        <h2>FLASK APP DEPLOYED TO AWS USING DOCKER</h4>
+    <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Illum officia asperiores impedit possimus quas 
+        officiis expedita velit, at architecto iusto natus modi quaerat, nulla laboriosam atque odio amet debitis 
+        et voluptas ea! Pariatur autem tempora placeat saepe doloribus minus ab maxime excepturi neque illo. Ratione,
+        sapiente magnam? Perspiciatis, molestias nihil.</p>
+    </div>
+    </div>   
+    </div> 
+</body>
+</html>
+```
+Now, let us run the flask app to see if it works. In the terminal, execute the command below:
+```bash
+set FLASK_APP='app.py'
+flask run -p 5000
+```
+You should have the information below in the terminal:
+```bash
+* Environment: production
+WARNING: This is a development server. Do not use it in a production deployment.
+Use a production WSGI server instead.
+* Debug mode: off
+* Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
+```
+Now head over to the browser on  `http://127.0.0.1:5000/` to see if the application is running.
+![App running locally](/engineering-education/using-ECS-to-deploy-a-docker-app-to-AWS/app-running.png)
+
+### Dockerizing the flask app
+Now that our application is up and running, we need to dockerize it. Dockerization is packaging the application and its environmental libraries and dependencies into one so that the app can run anywhere without having to perform new environmental configurations.
+
```

Comment:
You can also mention external link that details more about Dockers (something like a tutorial preferably)